pkg/fleets/controller_test.go

```diff
@@ -196,7 +196,9 @@ func TestControllerSyncFleet(t *testing.T) {
 	})
 
 	t.Run("gameserverset with different image details", func(t *testing.T) {
-		assert.NoError(t, utilruntime.ParseFeatures(string(utilruntime.FeaturePlayerTracking)+"=true"))
+		utilruntime.FeatureTestMutex.Lock()
```

Comment:
This was the only place in our test code that didn't use the feature test mutex, which could have caused a race condition.